common/src/test/java/tachyon/TachyonURITest.java

```diff
@@ -483,4 +483,14 @@ private void testParentChild(String target, String parent, String child) {
           new TachyonURI(new TachyonURI(parent), new TachyonURI(child)).toString());
     }
   }
+  
+  @Test
+  public void testGetLeadingPath() {
```

Comment:
The naming convention in Tachyon should be `getLeadingPathTest` for this test case.